package.json

```diff
@@ -49,7 +49,8 @@
     "lerna": "lerna"
   },
   "lint-staged": {
-    "*.{js,vue,ts}": "eslint"
+    "*.{js,vue,ts}": "eslint",
+    "**/i18n/*.csv": ["node ./core/scripts/utils/sort-translations.js", "git add"]
```

Comment:
NPM script should be launched there instead of pure `node ./....` so we can sort translations without lint staged